frontend/packages/lib-components/src/layout/flex-helpers.ts

```diff
@@ -11,14 +11,14 @@ interface FixedSpaceRowProps {
   justifyContent?: Property.JustifyContent
   alignItems?: Property.AlignItems
   marginBottom?: SpacingSize | string
-  wrap?: boolean
+  flexWrap?: boolean
```

Comment:
Oh we already have wrap prop here.. at some point we should probably remove it since I don't think it really works as intended. `FixedSpaceFlexWrap` and `AdaptiveFlex` are better options although also imperfect.

Reply:
But no need to do anything now